include/fmt/std.h

```diff
@@ -145,6 +145,26 @@ FMT_END_NAMESPACE
 #endif
 
 FMT_BEGIN_NAMESPACE
+FMT_EXPORT
+template <std::size_t N, typename Char> struct formatter<std::bitset<N>, Char> {
+  template <typename ParseContext>
+  FMT_CONSTEXPR auto parse(ParseContext& ctx) -> decltype(ctx.begin()) {
+    return ctx.begin();
+  }
+
+  template <typename FormatContext>
+  auto format(const std::bitset<N>& bs, FormatContext& ctx) const
+      -> decltype(ctx.out()) {
+    auto out = ctx.out();
+
+    for (auto pos = N; pos; --pos) {
```

Comment:
nit: Let's avoid an implicit conversion in the ctor to make this less surprising and avoid potential warnings:

```c++
for (auto pos = N; pos > 0; --pos) {
```

Reply:
Do you mean replacing contextual conversion to bool by comparing against zero? Because I'm not seeing any conversions on that line other than that at all.